src/coreclr/nativeaot/Directory.Build.props

```diff
@@ -66,7 +66,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <FeaturePerfTracing>false</FeaturePerfTracing>
-    <FeaturePerfTracing Condition="'$(TargetsWindows)' == 'true'">true</FeaturePerfTracing>
+    <FeaturePerfTracing Condition="'$(TargetsWindows)' == 'true' or ('$(TargetsUnix)' == 'true' and '$(TargetsOSX)' != 'true')">true</FeaturePerfTracing>
```

Comment:
Is this supposed to match the condition in `Microsoft.NETCore.Native.Unix.targets`? Not sure how/if this would actually manifest, but `TargetsOSX` is not quite the same as `_IsApplePlatform`, since that includes osx and ios-like.

Reply:
The intent is to exclude OSX support (for now) and I used the properties that were used respectively by each environment: build (Build props used elsewhere in that file) and the publish (_IsApplePlatform gets defined in the parent Microsoft.NETCore.Native.targets file and gets used in other locations in the targets file) environment.

Reply:
Note that we have people actively working on native AOT for iOS too.